src/providers/node.rs

```diff
@@ -123,7 +123,7 @@ impl NodeProvider {
 
     pub fn get_start_cmd(app: &App) -> Result<Option<String>> {
         if NodeProvider::has_script(app, "start")? {
-            return Ok(Some("npm run start".to_string()));
+            return Ok(Some("npm start".to_string()));
```

Comment:
Verbosity is better, I don't think this is a good change.

Reply:
Previously it was `npm start`. I am fine either ways.

Reply:
Agree with @Milo123459 that verbosity is better. Would prefer to stick with `npm run ...` versions